### PR TITLE
give-devices page: style improvement

### PR DESCRIPTION
**Pull request by theflucs**
Update of closed PR #171

### CODE REVIEW

File: pages/give-devices.js [hunk 1]
```diff
@@ -108,10 +108,10 @@ export default function GiveDevicesPage() {
       </Head>
       <Navbar />
       <main>
-        <div className="m-auto w-3/4 p-10 my-10 bg-green-300 border border-gray-300">
-          <div className="mt-10 sm:mt-0">
+        <div className="p-12">
```
Comment (RedFox0x20) [inline pages/give-devices.js:111]: Might be worth changing this to `sm:p-12` the reason being that on smaller devices (such as phones) this padding can be quite significant as it squashes the text together. Having no padding on these devices appears to work quite well and makes good use of the extra space given to it.

Noting: The updating of the CSS breakpoints does not appear to work properly on the Vercel host for me currently, however this should provide improved responsiveness.

**Review by RedFox0x20 — CHANGES_REQUESTED** [review]
Great stuff @theflucs !

This is much nicer on the eyes and provides a much more _modern_ design that is less blocky and provides good visual focal points to the user.

**Review by RedFox0x20 — APPROVED** [review]
Perfect, scales down to mobile devices better now. Thanks for the contribution!

**Review by marekrozmus — APPROVED** [review]
I love it  :) - I was thinking about some UI improvements while doing #173 but I don't have the skills for that ;) Hopefully there won't be much conflicts when merging both PRs.

**Review by audreyfeldroy — APPROVED** [review]
@theflucs Both the code and the page are beautiful! Nice work and I appreciate your eye for detail here so much.

**Pull request merged**